scripts/blueprints/workflows/os-arch.yaml

```diff
@@ -71,19 +71,20 @@ output:
             uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4
             with:
               fetch-depth: 1
-          # https://github.com/docker/setup-qemu-action
-          - name: Setup QEMU
-            uses: docker/setup-qemu-action@68827325e0b33c7199eb31dd4e31fbe9023e06e3 # v3
-            with:
-              platforms: all
-              # https://hub.docker.com/r/tonistiigi/binfmt
-              image: tonistiigi/binfmt:qemu-v6.2.0
           # https://github.com/docker/setup-buildx-action
           - name: Setup buildx
-            uses: docker/setup-buildx-action@d70bba72b1f3fd22344832f00baa16ece964efeb # v3
+            id: setup-buildx
+            uses: docker/setup-buildx-action@988b5a0280414f521da01fcc63a27aeeb4b104db # v3.6.1
             with:
               driver-opts: network=host
               install: true
+          # https://github.com/docker/setup-qemu-action
+          - name: Setup QEMU
```

Comment:
These changes just switch the ordering of the actions, right? Or am I missing something? What is the reasoning behind that??

Reply:
We do this in Flowzone as well, by running buildx setup first, we can check the outputs to see which platforms are natively supported. So the Setup QEMU step ends up looking like this with the new `if` condition added by the node script.
```yaml
          # https://github.com/docker/setup-qemu-action
          - name: Setup QEMU
            uses: docker/setup-qemu-action@49b3bc8e6bdd4a60e6116a5414239cba5943d3cf # v3.2.0
            with:
              platforms: all
              # https://hub.docker.com/r/tonistiigi/binfmt
              image: tonistiigi/binfmt:qemu-v8.0.4-33
            if: contains(steps.setup-buildx.outputs.platforms, 'linux/arm/v6') == false
```
The only reason we add the if condition in node and not the template is I don't know how to escape all those special characters in contract templates like this, while still substituting the arch.